include/eve/conditional.hpp

```diff
@@ -420,6 +517,16 @@ namespace eve
       EVE_ASSERT(b<=e, "[eve::keep_between] Index mismatch for begin/end");
     }
 
+    //==============================================================================================
+    //! @brief Extend a conditional expression with an laternative value
+    //!
+    //! By default, conditional operations use their unchanged input whenever the conditional
+    //! evaluates to `false`. By using this member function, operations will use the user provided
+    //! eve::value `v` instead.
+    //!
+    //! @param  v Value to use if the conditional expression evaluates to `false`
+    //! @return An instance of eve::or_ wrapping current conditional expression with `v`.
+    //==============================================================================================
```

Comment:
I dislike the copy-paste a lot.

Reply:
I know. There's some advanced Doyxgen tricks to limit it, but I was planning to do that later.
The doc here was meant so the conditional tutorial had working links.